qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -177,6 +180,29 @@ public void test70BindMountCustomPathConfAndJvmOptions() throws Exception {
         assertThat(nodesResponse, containsString("\"using_compressed_ordinary_object_pointers\":\"false\""));
     }
 
+    /**
+     * Check that the default config can be overridden using a bind mount, and that env vars are respected
+     */
+    public void test71BindMountCustomPathWithDifferentUID() throws Exception {
+//        copyFromContainer(installation.config("elasticsearch.yml"), tempDir.resolve("elasticsearch.yml"));
```

Comment:
Maybe remove this line now?

Reply:
Ugh 🤦‍♂

Reply:
Removed in f5aa265686d71f1fcbec53f3039b22bc83ca3364, thank you.